### DISCUSSION TITLE
Welcome to geostyler Discussions!

### DISCUSSION THREAD

**Discussion by KaiVolland** [geostyler/geostyler#1303] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rdewit** [comment]
Hi GeoStyler team and community, now that I've created a few PRs for `geostyler-openlayers-parser`, I think it's time to introduce myself and share how we would like to use GeoStyler and see it become successful.

I've been working with open source spatial tools and standards since about 2005 and have contributed a little to `OpenLayers 2` back in the day. Now, working for a company that values and relies heavily on open source and open standards, Pozi, I have the opportunity to contribute back again!

We deal with many different data sources and formats and would benefit from a unifying styling format 'to rule them all'. On top of that, we need an intuitive UI for managing these styles and GeoStyler ticks most of those boxes.

I appreciate how the GeoStyler team puts effort into creating a community and look forward to making GeoStyler an integral part of our product and keep contributing along the way!

**Comment by hebersgit** [comment]
Before I propose several pull requests and post an issue in the next few days, I would like to briefly introduce myself, as I have not yet been active in this project. I am Markus and I have been working as a software developer in the GIS field for over 25 years. I work for a company in Bonn, Germany (some of the people active here also seem to come from Bonn). We now have the task of converting QML styles into SLD-styles and later also into Mapbox-styles for an internal process. Geostyler basically fulfills this task well, but still has some gaps in the area of text styles and in the processing of size units. We have already implemented most of these in our local fork and would like to play them back in the form of pull requests.

**Reply by KaiVolland** [reply]
Hey @hebersgit. Every new participant is very welcome. Really looking forward to your contributions.
